tests/ethereum/main.rs

```diff
@@ -145,7 +145,9 @@ fn run_test_unit(path: &Path, unit: &TestUnit) {
                     assert!(exec_results[0].receipt.status.coerce_status());
                     // This is overly strict as we only need the newly created account's code to be empty.
                     // Extracting such account is unjustified complexity so let's live with this for now.
-                    assert!(exec_results[0].state.values().all(|account| account.info.is_empty_code_hash()));
+                    assert!(exec_results[0].state.values().all(|account| account
+                        .as_ref()
+                        .map_or(false, |account| account.basic.code.is_empty())));
```

Comment:
This should be `true`? Existing accounts getting self-destructed doesn't conflict with the condition that all changed accounts must not have code attached.

If `false` is correct then `is_some_and` should read better.

Reply:
yes, `true` here makes sense